src/main/resources/python/generator/writer.py

```
@@ -0,0 +1,82 @@
+    def get_math_methods(self, class_name: str) -> list[str]:
+        methods = []
+        ops = ['add', 'sub', 'mul', 'truediv', 'floordiv', 'mod', 'pow']
+        for op in ops:
+            methods.append(f"    def __{op}__(self, other: Any) -> Any: ...")
+            methods.append(f"    def __r{op}__(self, other: Any) -> Any: ...")
+            methods.append(f"    def __i{op}__(self, other: Any) -> Any: ...")
+
+        for op in ['neg', 'pos', 'abs', 'invert']:
+            methods.append(f"    def __{op}__(self) -> '{class_name}': ...")
+
+        methods.append(f"    def __eq__(self, other: Any) -> bool: ...")
```

Ruff (0.15.0)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)